innsending/src/test/kotlin/no.nav.helsearbeidsgiver.inntektsmelding.innsending/ForespoerselUtils.kt

```diff
@@ -0,0 +1,48 @@
+package no.nav.helsearbeidsgiver.inntektsmelding.innsending
+
+import no.nav.helsearbeidsgiver.felles.domene.Forespoersel
+import no.nav.helsearbeidsgiver.felles.domene.ForespurtData
+import no.nav.helsearbeidsgiver.felles.domene.ForslagInntekt
+import no.nav.helsearbeidsgiver.felles.domene.ForslagRefusjon
+
+fun Forespoersel.utenPaakrevdAGP(): Forespoersel =
+    copy(
+        forespurtData =
+            forespurtData.copy(
+                arbeidsgiverperiode =
+                    ForespurtData.Arbeidsgiverperiode(
+                        paakrevd = false,
+                    ),
+            ),
+    )
+
+fun Forespoersel.utenPaakrevdInntekt(): Forespoersel =
+    copy(
+        forespurtData =
+            forespurtData.copy(
+                inntekt =
+                    ForespurtData.Inntekt(
+                        paakrevd = false,
+                        forslag =
+                            ForslagInntekt.Fastsatt(
+                                fastsattInntekt = 8795.0,
```

Comment:
Hvor kommer det tallet her fra? 🤔

Reply:
I prod? Hvis Spleis ikke trenger at AG oppgir inntekt så sender de HAG en inntekt som vi bare sender tilbake til dem. Spleis krever at vi har med inntekt når vi sender inntektsmeldingen til dem, så dette er en workaround for det.